tests/functional/codegen/features/test_constructor.py

```diff
@@ -101,15 +101,15 @@ def foo() -> int128:
     return 5
     """
 
-    get_contract_with_gas_estimation(large_input_code_2, *[17])
+    get_contract(large_input_code_2, *[17])
 
-    with pytest.raises(TypeError):
-        get_contract_with_gas_estimation(large_input_code_2, *[2**130])
+    with pytest.raises(EncodeError):
+        get_contract(large_input_code_2, *[2**130])
```

Comment:
hmm probably out of scope of this PR but this is just testing the abi of the ctor, not the clamper in the actual bytecode. can we add a comment?

Reply:
I'll fix it